app/entity_helper.py

```diff
@@ -0,0 +1,84 @@
+from dataclasses import is_dataclass
+from datetime import datetime
+from typing import Any
+from functools import wraps
+
+
+class EntityHelper:
+    __placeholder_symbol = "?"
+
+    def __init__(self, entity_type: type, table_name: str) -> None:
+        if not is_dataclass(entity_type):
+            raise ValueError(f"EntityHelper is able to work with "
+                             f"dataclasses only. {type} is not a dataclass")
+        self.__entity_type: type = entity_type
+        self.__table_name: str = table_name
+        self.__entity_type_fields: tuple = tuple(
+            self.__entity_type
+            .__dataclass_fields__
+            .keys()
+        )
+        self.__insert_into_statement: str = (
+            f"insert into '{self.__table_name}' "
+            f"{self.__entity_type_fields[1:]} values "
+            f"({self.make_placeholders(len(self.__entity_type_fields) - 1)})"
+        )
+        self.__select_all_statement: str = (
+            f"select * from {self.__table_name}"
```

Comment:
Using uppercase for SQL commands (e.g., SELECT, INSERT, UPDATE) has been a common convention and is often recommended in many coding style guides. This convention helps differentiate SQL keywords from table and column names, making the code more readable and easier to understand

Reply:
yeah, I know, I'm just too old to do it like this 🤪 I do use this approach of using uppercase for sql commands when I when I write a separate *.sql files, but polluting my source code with UPPERCASE - it is something I will avoid, unless it's a company policy to write it like this. Uppercased constant values, `defined` (well-known) literals and sql statements - it is something that makes our code smell. So, yeah, I never use this convention inside my non-sql files for that reason